Tests fail for the new installation

After installing the latest version ([this commit](https://github.com/sobolevn/webpack/commit/0095ebad27291b38154d38b087ebf785dc873236)) I have faced an issue with tests. After I try to run `npm run tests` they fail.

I have figured out, that this lines ([0](https://github.com/vuejs-templates/webpack/blob/0095ebad27291b38154d38b087ebf785dc873236/template/test/unit/specs/Hello.spec.js#L10), [1](https://github.com/vuejs-templates/webpack/blob/0095ebad27291b38154d38b087ebf785dc873236/template/test/e2e/specs/test.js#L15), [2](https://github.com/vuejs-templates/webpack/blob/0095ebad27291b38154d38b087ebf785dc873236/template/test/e2e/specs/test.js#L16)) would never evaluate to `true`, since the logic is completely different.

I have fixed it for me, sending a PR. Now the tests pass. 

> TOTAL: 1 SUCCESS
> 
> 
> =============================== Coverage summary ===============================
> Statements   : 100% ( 9/9 )
> Branches     : 100% ( 4/4 ), 1 ignored
> Functions    : 100% ( 2/2 )
> Lines        : 100% ( 4/4 )
> ================================================================================
> 
> 
>  ✔ Element <#app> was visible after 83 milliseconds.
>  ✔ Testing if element <.hello> is present.
>  ✔ Testing if element h1 contains text: "Welcome to Your Vue.js App".
>  ✔ Testing if element img has count: 1
> OK. 4 assertions passed. (5.059s)

Here's the full output of the problem:

> 
> [19:58:03] sobolev :: MacBook-Pro-Nikita  ➜  Documents/Frontends/razgrebaem » npm run test
> 
> > razgrebaem@1.0.0 test /Users/sobolev/Documents/Frontends/razgrebaem
> > npm run unit && npm run e2e
> 
> 
> > razgrebaem@1.0.0 unit /Users/sobolev/Documents/Frontends/razgrebaem
> > karma start test/unit/karma.conf.js --single-run
> 
> 17 10 2016 20:01:39.118:INFO [karma]: Karma v1.3.0 server started at http://localhost:9876/
> 17 10 2016 20:01:39.121:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
> 17 10 2016 20:01:39.126:INFO [launcher]: Starting browser PhantomJS
> 17 10 2016 20:01:39.781:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket /#YKrlL3oAoyCWsPOWAAAA with id 32138849
> 
>   Hello.vue
>     ✗ should render correct contents
> 	expected 'Welcome to Your Vue.js App' to equal 'Hello Vue!'
> 
> 
> PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 1 of 1 (1 FAILED) ERROR (0.017 secs / 0.01 secs)
> 
> 
> =============================== Coverage summary ===============================
> Statements   : 100% ( 9/9 )
> Branches     : 100% ( 4/4 ), 1 ignored
> Functions    : 100% ( 2/2 )
> Lines        : 100% ( 4/4 )
> ================================================================================
> 
> npm ERR! Darwin 15.6.0
> npm ERR! argv "/usr/local/Cellar/node/5.5.0/bin/node" "/usr/local/bin/npm" "run" "unit"
> npm ERR! node v5.5.0
> npm ERR! npm  v3.5.3
> npm ERR! code ELIFECYCLE
> npm ERR! razgrebaem@1.0.0 unit: `karma start test/unit/karma.conf.js --single-run`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the razgrebaem@1.0.0 unit script 'karma start test/unit/karma.conf.js --single-run'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the razgrebaem package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     karma start test/unit/karma.conf.js --single-run
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs razgrebaem
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls razgrebaem
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     /Users/sobolev/Documents/Frontends/razgrebaem/npm-debug.log
> 
> npm ERR! Darwin 15.6.0
> npm ERR! argv "/usr/local/Cellar/node/5.5.0/bin/node" "/usr/local/bin/npm" "run" "test"
> npm ERR! node v5.5.0
> npm ERR! npm  v3.5.3
> npm ERR! code ELIFECYCLE
> npm ERR! razgrebaem@1.0.0 test: `npm run unit && npm run e2e`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the razgrebaem@1.0.0 test script 'npm run unit && npm run e2e'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the razgrebaem package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     npm run unit && npm run e2e
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs razgrebaem
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls razgrebaem
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     /Users/sobolev/Documents/Frontends/razgrebaem/npm-debug.log

--- a/template/test/e2e/specs/test.js
+++ b/template/test/e2e/specs/test.js
@@ -11,9 +11,9 @@ module.exports = {
     browser
       .url(devServer)
       .waitForElementVisible('#app', 5000)
-      .assert.elementPresent('.logo')
-      .assert.containsText('h1', 'Hello Vue!')
-      .assert.elementCount('p', 3)
+      .assert.elementPresent('.hello')
+      .assert.containsText('h1', 'Welcome to Your Vue.js App')
+      .assert.elementCount('img', 1)
       .end(){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
   }{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
 }{{#if_eq lintConfig "airbnb"}};{{/if_eq}}

--- a/template/test/unit/specs/Hello.spec.js
+++ b/template/test/unit/specs/Hello.spec.js
@@ -7,6 +7,7 @@ describe('Hello.vue', () => {
       el: document.createElement('div'),
       render: (h) => h(Hello){{#if_eq lintConfig "airbnb"}},{{/if_eq}}
     }){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
-    expect(vm.$el.querySelector('.hello h1').textContent).to.equal('Hello Vue!'){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+    expect(vm.$el.querySelector('.hello h1').textContent)
+      .to.equal('Welcome to Your Vue.js App'){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
   }){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
 }){{#if_eq lintConfig "airbnb"}};{{/if_eq}}

